rmf_task_msgs/msg/TaskSummary.msg

```diff
@@ -8,6 +8,9 @@ string task_id
 
 TaskProfile task_profile
 
+# current task phases
+Phase[] phases
```

Comment:
I think it's a bit confusing to have `pending_phases` and `completed_phases` inside of a message called `Phase`.

Maybe we should rename the message type to `PhaseTier`, and we could make this field into `PhaseTier[] tier`.